content/en/about.md

```diff
@@ -38,16 +36,19 @@ Emeritus:
 
 ## Teams
 
-The NumPy project is growing! &#127881; We have teams for:
+The NumPy project leadership is actively working on diversifying contribution pathways to the project.<br>
+NumPy currently has the following teams:
 
-- code
+- developer
```

Comment:
```suggestion
- development
```

Reply:
I saw that, but I thought the names corresponded to GitHub team names.

Reply:
I guess not! https://github.com/orgs/numpy/teams